src/create-glamorous.js

```diff
@@ -94,13 +94,13 @@ function createGlamorous(splitProps) {
             Object.freeze(this.state.theme)
 
           // create className to apply
-          const fullClassName = getGlamorClassName(
```

Comment:
Just turning this into an object because there are a lot of args (and I'm going to add yet another)